README.md

```diff
@@ -1 +1,4 @@
-# code-review.
\ No newline at end of file
+# code-review.
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
```

Comment:
hey, great work so far but please use a more descriptive Readme,I would suggest you use the Readme template provided for you to make your work more professional